cloudslang-cli/pom.xml

```diff
@@ -149,7 +149,7 @@
 							<id>cslang</id>
 						</program>
 					</programs>
-                    <extraJvmArguments>-Dpython.path=@BASEDIR@/python-lib</extraJvmArguments>
+                    <extraJvmArguments>-XX:PermSize=128m -XX:MaxPermSize=256m -Dpython.path=@BASEDIR@/python-lib</extraJvmArguments>
```

Comment:
The issue you are trying to solve is in the CI (not regular usage of CLI)
It does not make sense to change CLI config , it makes more sense to try first to change CI (CircleCI) configuration....